src/ui/hooks/useDynamicLoadingMessage.ts

```diff
@@ -53,6 +53,11 @@ export function useDynamicLoadingMessage(
   stalledTimeout: number
 ) {
   const [message, setMessage] = useState(isProcessed ? initialMessage : "Processing...");
+  const [additionalInfo, setAdditionalInfo] = useState(
+    isProcessed
+      ? ""
+      : "This could take a while, depending on the complexity and length of the replay."
+  );
```

Comment:
This does not need to be in state. It's never updated. It's just a value derived from a prop.